velox/exec/SetAccumulator.h

```diff
@@ -106,10 +153,50 @@ struct SetAccumulator {
                                  : uniqueValues.size();
   }
 
+  void serializeNullIndex(char* buffer) {
+    auto nullIndexValue =
+        nullIndex.has_value() ? nullIndex.value() : kNoNullIndex;
+    memcpy(buffer, &nullIndexValue, kVectorSizeT);
```

Comment:
It would be nice to extract all these memcpy into `write<T>` APIs or reuse something like ByteOutputStream.